src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/get_indices.test.ts

```diff
@@ -91,16 +129,22 @@ describe('getIndices', () => {
         },
       ],
     };
-    expect(responseToItemArray(result, mockGetTags)).toMatchSnapshot();
-    expect(responseToItemArray({}, mockGetTags)).toEqual([]);
+    expect(responseToItemArray(result, getIndexTags)).toMatchSnapshot();
+    expect(responseToItemArray({}, getIndexTags)).toEqual([]);
+  });
+
+  it('matched items are deduped', () => {
+    const setA = [{ name: 'a' }, { name: 'b' }] as MatchedItem[];
+    const setB = [{ name: 'b' }, { name: 'c' }] as MatchedItem[];
+    expect(dedupeMatchedItems(setA, setB)).toHaveLength(3);
```

Comment:
nit: It might be nice to add just a couple more assertions here -- the `dedupeMatchedItems` function is a bit hard to read and more test examples would make it clearer how exactly it's working.